diff --git a/config/config.yml b/config/config.yml
index 4d5c37f..73ebcce 100644
--- a/config/config.yml
+++ b/config/config.yml
@@ -9,7 +9,7 @@ model:
   n_ffn: 1024
   n_layer: 8
   patch_size: 20
-  n_classes: 1
+  n_classes: 2
   resolution: [600, 400]
   scale_init: 0
   mlp_head: True
diff --git a/train.py b/train.py
index 70198b4..781784e 100644
--- a/train.py
+++ b/train.py
@@ -75,6 +75,7 @@ def main(args=None):
     config = RWKVConfig()
     if args is not None:
         config.from_yaml(args.config)
+    print(config)
     # Safety first
     # should reduce unnecessary padding
     config.ctx_len = int(
@@ -95,9 +96,10 @@ def main(args=None):
 
     # get classes
     config.num_classes = train_dataset.get_classes()
+    print("Classes:", config.num_classes)
 
-    # model = FaceRWKV(config)
-    model = CNNClassifier(train_dataset.get_classes())
+    model = FaceRWKV(config)
+    # model = CNNClassifier(train_dataset.get_classes())
 
     CUDA = torch.cuda.is_available()
     if CUDA:
diff --git a/wandb/debug-internal.log b/wandb/debug-internal.log
index 6f746a2..cbd8b6a 120000
--- a/wandb/debug-internal.log
+++ b/wandb/debug-internal.log
@@ -1 +1 @@
-run-20240221_225358-56vxbyvt/logs/debug-internal.log
\ No newline at end of file
+run-20240222_195836-fm1tbvw7/logs/debug-internal.log
\ No newline at end of file
diff --git a/wandb/debug.log b/wandb/debug.log
index 8dc03d6..459bdbe 120000
--- a/wandb/debug.log
+++ b/wandb/debug.log
@@ -1 +1 @@
-run-20240221_225358-56vxbyvt/logs/debug.log
\ No newline at end of file
+run-20240222_195836-fm1tbvw7/logs/debug.log
\ No newline at end of file
diff --git a/wandb/latest-run b/wandb/latest-run
index 1c3e872..5bff54f 120000
--- a/wandb/latest-run
+++ b/wandb/latest-run
@@ -1 +1 @@
-run-20240221_225358-56vxbyvt
\ No newline at end of file
+run-20240222_195836-fm1tbvw7
\ No newline at end of file
